webcompat/form.py

```diff
@@ -187,15 +187,17 @@ def build_formdata(form_object):
     Here's a description of what the Issues API expects to create an issue
     (as of March 25, 2014):
 
```

Comment:
nit: We probably want to change the date here. Or just remove the date entirely.